cmd/kube-scheduler/app/server.go

```diff
@@ -79,15 +79,14 @@ suitable Node. Multiple different schedulers may be used within a cluster;
 kube-scheduler is the reference implementation.
 See [scheduling](https://kubernetes.io/docs/concepts/scheduling-eviction/)
 for more information about scheduling and the kube-scheduler component.`,
-		Run: func(cmd *cobra.Command, args []string) {
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I suggest to not change the error messages. Just return the errors to preserve the current behavior, whatever it is.

Reply:
Fixed thanks

Reply:
You may also want to set `SilenceErrors` and `SilenceUsage` to avoid Cobra printing the error and the usage information on the error.

Reply:
Ok, fixed. 
As you said, if we don't set SilenceErrors/SilenceUsage, it seems that cobra changes error msg and adds usage message on error msg. thanks

https://github.com/spf13/cobra/blob/v1.2.1/command.go#L975-L994